cdk/lib/rds-stack.ts

```diff
@@ -38,7 +46,7 @@ export class RDSStack extends Stack {
         vpc: props.vpc,
         credentials: {
           username: this.rdsDbUser,
-          password: this.secret.secretValue,
+          password: this.rdsPassword.secretValue,
```

Comment:
does this need .toString() I saw in api-stack.ts it had .secretValue.toString() appended to it?

Reply:
Nay, the RDS construct takes an ISecret or something, according to the typing on the constructor for RDS.  The Lambda constructor takes a string tho, feels like the two teams weren't on the same page.